.mergify.yml

```diff
@@ -1,7 +1,13 @@
 queue_rules:
   - name: default
-    conditions:
+    queue_conditions:
+      - check-success=buildkite/elastic-package
+      - check-success=CLA
+      - base=main
+      - author~=^dependabot(|-preview)\[bot\]$
+    merge_conditions:
       - check-success=buildkite/elastic-package
+    merge_method: squash
 
 pull_request_rules:
   - name: automatic approval for Dependabot pull requests
```

Comment:
Well, is the condition of this rule needed now?

Reply:
> Do you mean the `automatic approval for Dependabot pull requests` pull request rule?
> That rule does not have any condition.

Yes, I refer to this one.

It has this condition, right?
```
    conditions:
      - author~=^dependabot(|-preview)\[bot\]$
```

Is it needed? Or why was it removed from the other rule and not from here? :thinking:

Reply:
Ok! I didn't see it because Github didn't show it (it was collapsed) :sweat_smile: 

What I understand is that from the other rule it is removed because it uses as an action to enqueue the PR (and it uses the conditions defined in the queue). In this rule, there is no action to enqueue the PR. This is just to approve the PR.

What I'm wondering now if it is required to add as part of the `queue_conditions` this `#approved-reviews-by` one ([docs](https://docs.mergify.com/configuration/conditions/#attributes-list)), but it's true that before there was no such a rule:
```yaml
    queue_conditions:
      - check-success=buildkite/elastic-package
      - check-success=CLA
      - base=main
      - author~=^dependabot(|-preview)\[bot\]$
      - "#approved-reviews-by >= 1"
```

Reply:
> What I understand is that from the other rule it is removed because it uses as an action to enqueue the PR (and it uses the conditions defined in the queue). In this rule, there is no action to enqueue the PR. This is just to approve the PR.

Ah ok, this is what I was not understanding.

> What I'm wondering now if it is required to add as part of the queue_conditions this #approved-reviews-by one ([docs](https://docs.mergify.com/configuration/conditions/#attributes-list))

I guess we are checking this with the branch protection rules?

Reply:
> I guess we are checking this with the branch protection rules?
Yes, I think so, that rule is there.